### DISCUSSION TITLE
Blank page instead of admin console

### DISCUSSION THREAD

**Discussion by faermanj** [keycloak/keycloak#12136] [Q&A]
Hello,

I managed to get keycloak started and opening the welcome page, using docker on gitpod.io.
However, the admin console does not open, just displays a blank page.
No error show up on the console and all requests are returning 200.
Here is a quick demo of the issue: https://vimeo.com/712039611

Thanks in advance for any help.

**Comment by abstractj** [comment]
Hi @faermanj, I'm not familiar with gitpod, but it would be nice to share the YAML file, if possible. In this way, others can give it a try and help you.

**Reply by faermanj** [reply]
Sure, here it is gitpod yaml
https://github.com/CaravanaCloud/red-pod/blob/main/.gitpod.yml

**Comment by HoussemGargouri** [comment]
Hello, 
Same problem for me too ! Did you solve it pls? thx

**Reply by faermanj** [reply]
@HoussemGargouri Yes, i believe i was missing KC_PROXY=true
Here is how i got it working:
https://github.com/CaravanaCloud/red-pod/blob/main/.gitpod/docker-compose-gitpod.yml
